scripts/update/Updater.php

```diff
@@ -315,6 +315,10 @@ public function update($currentVersion)
             call_user_func(new RegisterPciAudioRecording(), ['0.4.0']);
             $this->setVersion('4.6.0');
         }
-        $this->skip('4.6.0', '4.6.5');
+        $this->skip('4.6.0', '4.7.0');
+
+        if ($this->isVersion('4.7.0')) {
```

Comment:
The update need to be applied for installs in version `4.6.5` to version `4.7.0`, so instead it should be something like : 

```php
 $this->skip('4.6.0', '4.6.5');
if ($this->isVersion('4.6.5')) {
    call_user_func(new RegisterPciAudioRecording(), ['0.5.0']);
    $this->setVersion('4.7.0');
}